CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake
                               ${CMAKE_SOURCE_DIR}/cmake/math
 )
 include(Psi4Macros)
-no_in_source()
+include(autocmake_safeguards)
```

Comment:
This line is causing me problems:

    % cmake ~/Code/Chemistry/psi4
    CMake Error at cmake/autocmake_safeguards.cmake:25 (message):
      Unknown build type "".  Allowed values are Debug, Release, RelWithDebInfo
      (case-insensitive).
    Call Stack (most recent call first):
     CMakeLists.txt:34 (include)

Not sure where it should go to fix it. If I specify ```-DCMAKE_BUILD_TYPE=Debug``` it works. Reading the options below it should defaulted to Release.